spring-cloud-aws-sns/src/main/java/io/awspring/cloud/sns/sms/attributes/ADM.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.sns.sms.attributes;
+
+public class ADM {
+	private Long ttl;
+
+	public Long getTtl() {
+		return ttl;
+	}
+
+	public void setTtl(Long ttl) {
+		this.ttl = ttl;
+	}
+
+	public static Builder builder() {
+		return new Builder();
+	}
+
+	public static final class Builder {
+		private Long ttl;
+
+		private Builder() {
+		}
+
+		public Builder withTtl(Long ttl) {
```

Comment:
For the sake of consistency with other builder classes in the project and builder classes in AWS SDK, instead of "withers" method names should be just named with an attribute"

```
Builder ttl(Long ttl) {
...
}
```